src/it/resources/expected/extern_generics/cppcli/MyRecord.cpp

```diff
@@ -0,0 +1,60 @@
+// AUTOGENERATED FILE - DO NOT MODIFY!
+// This file was generated by Djinni from extern_generics.djinni
+
+#include "MyRecord.hpp"  // my header
```

Comment:
the `// my header` comment is generated and expected?

Reply:
hmm that's what the generator produces, you can see it in all the other generated cppcli sources in expected.

Reply:
oh, indeed, its in the code 
https://github.com/cross-language-cpp/djinni-generator/blob/4649bd1ed78dec54a81be0ee563cbc9b02be6931/src/main/scala/djinni/generator.scala#L611
lol 
so lets leaf that (for now)